Source/WebCore/animation/WebAnimationTypes.h

```diff
@@ -64,6 +67,33 @@ using AnimationEvents = Vector<Ref<AnimationEventBase>>;
 using PropertyToTransitionMap = HashMap<CSSPropertyID, RefPtr<CSSTransition>>;
 using CSSAnimationCollection = ListHashSet<RefPtr<CSSAnimation>>;
 
+using AnimatableProperty = std::variant<CSSPropertyID, AtomString>;
+using AnimatablePropertyToTransitionMap = HashMap<AnimatableProperty, RefPtr<CSSTransition>>;
```

Comment:
Why not `Ref<>` instead of `RefPtr<>`? Can map values be null?

Reply:
I'll look into that as a follow-up since this is a broader change in the places where `AnimatablePropertyToTransitionMap` will eventually be used (in replacement of the existing `PropertyToTransitionMap`). I filed https://bugs.webkit.org/show_bug.cgi?id=249530 to track that work.

Reply:
There's a patch up for review in https://github.com/WebKit/WebKit/pull/8361.